tools/go-cli/go-whisk-cli/commands/action.go

```diff
@@ -662,29 +662,27 @@ func parseAction(cmd *cobra.Command, args []string) (*whisk.Action, bool, error)
-
-  actionUpdateCmd.Flags().BoolVar(&flags.action.docker, "docker", false, "treat artifact as docker image path on dockerhub")
-  actionUpdateCmd.Flags().BoolVar(&flags.action.copy, "copy", false, "treat artifact as the name of an existing action")
-  actionUpdateCmd.Flags().BoolVar(&flags.action.sequence, "sequence", false, "treat artifact as comma separated sequence of actions to invoke")
-  actionUpdateCmd.Flags().StringVar(&flags.action.kind, "kind", "", "the kind of the action runtime (example: swift:3, nodejs:6)")
-  actionUpdateCmd.Flags().StringVar(&flags.action.shared, "shared", "", "shared action (default: private)")
-  actionUpdateCmd.Flags().StringVar(&flags.action.xPackage, "package", "", "package")
-  actionUpdateCmd.Flags().IntVarP(&flags.action.timeout, "timeout", "t", -1, "the timeout limit in milliseconds when the action will be terminated")
-  actionUpdateCmd.Flags().IntVarP(&flags.action.memory, "memory", "m", -1, "the memory limit in MB of the container that runs the action")
-  actionUpdateCmd.Flags().IntVarP(&flags.action.logsize, "logsize", "l", -1, "the logsize limit in MB of the container that runs the action")
-  actionUpdateCmd.Flags().StringSliceVarP(&flags.common.annotation, "annotation", "a", []string{}, "annotations")
-  actionUpdateCmd.Flags().StringSliceVarP(&flags.common.param, "param", "p", []string{}, "default parameters")
+    actionCreateCmd.Flags().BoolVar(&flags.action.docker, "docker", false, "treat ACTION as docker image path on dockerhub")
+    actionCreateCmd.Flags().BoolVar(&flags.action.copy, "copy", false, "treat ACTION as the name of an existing action")
+    actionCreateCmd.Flags().BoolVar(&flags.action.sequence, "sequence", false, "treat ACTION as comma separated sequence of actions to invoke")
+    actionCreateCmd.Flags().StringVar(&flags.action.kind, "kind", "", "the `KIND` of the action runtime (example: swift:3, nodejs:6)")
+    actionCreateCmd.Flags().StringVar(&flags.action.shared, "shared", "no", "action visibility `SCOPE`; yes = shared, no = private")
+    actionCreateCmd.Flags().IntVarP(&flags.action.timeout, "timeout", "t", -1, "the timeout `LIMIT` in milliseconds when the action will be terminated")
+    actionCreateCmd.Flags().IntVarP(&flags.action.memory, "memory", "m", -1, "the memory `LIMIT` in MB of the container that runs the action")
+    actionCreateCmd.Flags().IntVarP(&flags.action.logsize, "logsize", "l", -1, "the logsize `LIMIT` in MB of the container that runs the action (default 10MB)")
+    actionCreateCmd.Flags().StringSliceVarP(&flags.common.annotation, "annotation", "a", nil, "`KEY VALUE` annotation pair(s)")
+    actionCreateCmd.Flags().StringSliceVarP(&flags.common.param, "param", "p", nil, "default action `KEY VALUE` parameter pair(s)")
+
+    actionUpdateCmd.Flags().BoolVar(&flags.action.docker, "docker", false, "treat artifact as docker image path on dockerhub")
```

Comment:
Should the flag descriptions for other commands also be update so that keywords are capitalized?